# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: accessible/tests/browser/hittest/browser_test_general.js
# Commit: 4570141f12e9
# Full Hash: 4570141f12e9018910dcff1b57fc8070a8601ede
# Author: Morgan Rae Reschenberg <mreschenberg@mozilla.com>
# Date: 2023-06-29 21:48:38
# Description:
#   Bug 1840343: Track containers and text leaves separately when fuzzy hittesting r=Jamie
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D182269
# ==============================================================================

diff -r 7569a78f5815 -r 4570141f12e9 accessible/tests/browser/hittest/browser_test_general.js
--- a/accessible/tests/browser/hittest/browser_test_general.js	Thu Jun 29 18:01:25 2023 +0000
+++ b/accessible/tests/browser/hittest/browser_test_general.js	Thu Jun 29 18:09:46 2023 +0000
@@ -321,7 +321,7 @@
  */
 addAccessibleTask(
   `
-<div id="generic"><span aria-hidden="true" id="visible">Mozilla</span><span id="invisible" style="display: block !important;border: 0 !important;clip: rect(0 0 0 0) !important;height: 1px !important;margin: -1px !important;overflow: hidden !important;padding: 0 !important;position: absolute !important;white-space: nowrap !important;width: 1px !important;">Mozilla</span><br>I am some other text</div>`,
+<div id="generic"><span aria-hidden="true" id="visible">Mozilla</span><span id="invisible" style="display: block !important;border: 0 !important;clip: rect(0 0 0 0) !important;height: 1px !important;margin: -1px !important;overflow: hidden !important;padding: 0 !important;position: absolute !important;white-space: nowrap !important;width: 1px !important;">hello world<br><div id="extraContainer">Mozilla</div></span><br>I am some other text</div>`,
   async function (browser, docAcc) {
     const generic = findAccessibleChildByID(docAcc, "generic");
     const invisible = findAccessibleChildByID(docAcc, "invisible");
